smartsim/wlm/slurm.py

```diff
@@ -253,6 +253,8 @@ def _get_alloc_cmd(nodes: int,
                     salloc_args += [prefix + opt, str(val)]
                 else:
                     salloc_args += ["=".join((prefix + opt, str(val)))]
+        else: 
+            raise SSConfigError("Expecting time, nodes, account as an argument")
```

Comment:
Incredibly minor point: The `SSConfigError` tends to be used for when smartsim cannot find `redis-server`, `redisai.so`, one of the ml backends, or some other binary file, and is usually thrown when a user is trying to run a an experiment, but forgot to run `smart build` or set their `SMARTSIM_DEP_INSTALL_PATH` or `RAI_PATH` env var.

I might suggest changing to a plain old `ValueError` (or maybe a `KeyError`, but that is kind of a stretch on the expected use of `KeyError`)